src/test/sighash_tests.cpp

```diff
@@ -125,10 +125,11 @@ BOOST_AUTO_TEST_CASE(sighash_test)
     std::cout << "[\n";
     std::cout << "\t[\"raw_transaction, script, input_index, hashType, signature_hash (result)\"],\n";
     #endif
-    int nRandomTests = 50000;
 
     #if defined(PRINT_SIGHASH_JSON)
-    nRandomTests = 500;
+    int nRandomTests = 500;
+    #else
+    int nRandomTests = 50000;
     #endif
     for (int i=0; i<nRandomTests; i++) {
```

Comment:
@paveljanik The `ifdef` was added in 232aa9e034d4d48f4d16dfdc6bc0d56c9274cfd3 by @maraoz to control if `data/sighash.json` test data should be written. Is it no longer needed?